Controllers/MedewerkerController.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
+
+
+
+namespace Backend.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+[Produces("application/json")]
+public class MedewerkerController : ControllerBase
+{
+    private readonly DatabaseContext _context;
+
+
+    public MedewerkerController(DatabaseContext context)
+    {
+        _context = context;
+    }
+
+[HttpGet]
+[Route("voorstelling/")]
```

Comment:
Waarom gebruik je hier medewerkercontroller? Er is al een controller hiervoor

Reply:
Ik wilde een controller die alle voorstellingEvents kan ophalen zonder tussenstappen.

Reply:
Maak dan een event controller